src/app/dashboard/page.tsx

```
@@ -1,13 +1,14 @@
+import Link from 'next/link';
 import { createClientComponentClient } from '@supabase/auth-helpers-nextjs';
 import { useRouter } from 'next/navigation';
 import ChatInterface from '../components/chat/ChatInterface';
-import { 
-  Sparkles, Moon, BookOpen, Compass, 
-  Heart, LogOut, Menu, X, 
-  Feather
+import {
+  Sparkles, Moon, BookOpen, Compass,
+  Heart, LogOut, Menu, X,
+  Feather, Map
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)